equinox/nn/_attention.py

```diff
@@ -179,8 +211,44 @@ def __init__(
         - `dtype`: The dtype to use for all trainable parameters in this layer.
             Defaults to either `jax.numpy.float32` or `jax.numpy.float64` depending
             on whether JAX is in 64-bit mode.
+        - `kv_cache`: The callable module to handle KV caching.
         - `key`: A `jax.random.PRNGKey` used to provide randomness for parameter
             initialisation. (Keyword only argument.)
+
+
+
+        !!! info
+
+        The following shows an example of how to use `MultiheadAttention` for
+        autoregressive decoding with a `kv_cache` and `state`:
```

Comment:
Can you add a link to `StandardKVCache` here? You can do this with specifically the syntax ``[`equinox.nn.StandardKVCache`][]``.